requirements_doc.txt

```diff
@@ -8,7 +8,7 @@ memory_profiler
 neo
 seaborn!=0.11.2
 sphinx_copybutton
-mne-bids
+https://github.com/mne-tools/mne-bids/zipball/31a870e01fa03810cae3463f0c56201ae4de6584
```

Comment:
This fixes https://app.circleci.com/pipelines/github/mne-tools/mne-python/12400/workflows/d14bab05-80a0-4634-9d76-bcc8d0951bfb/jobs/40613 by using the commit from https://github.com/mne-tools/mne-bids/pull/944 until MNE-BIDS pushes out a new release